Woof/WoofTests/ViewModel/OwnerMainTabViewModelTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+final class OwnerMainTabViewModelTests: XCTestCase {
+    func testResetCurrentRoleMethodExistsInAPI() {
+        OwnerMainTabViewModel().resetCurrentRole()
+    }
+
+    func testResetCurrentRoleSuccessfullyChangesUserRoleToNone() {
```

Comment:
nit:

Keep it short:
```suggestion
    func testResetCurrentRoleChangesUserRoleToNone() {
```